librz/core/cmd/cmd_print.c

```diff
@@ -1275,6 +1276,16 @@ static void cmd_print_fromage(RzCore *core, const char *input, const ut8 *data,
 			free(s);
 		}
 	} break;
+	case 'A': // "pFA"
+	{
+		char *s = rz_axml_decode(data, size);
+		if (s) {
+			rz_cons_printf("%s", s);
+			free(s);
+		} else {
+			eprintf("Malformed object: did you supply enough data?\ntry to change the block size (see b?)\n");
```

Comment:
RZ_LOG?

Reply:
it's used in many other places in the file. Leave it this way for now.